.github/workflows/release-docker.yml

```diff
@@ -14,13 +14,15 @@ jobs:
     environment: 'prod'
     strategy:
       matrix:
-        cuda_version: ['12.6.1', '12.8.1']
+        cuda_version: ['12.6.1', '12.8.1', '12.9.1']
```

Comment:
In the follow-up, only cu129+all will be kept. All others will be removed. It should work on cu124, cu126, cu128, and cu129. hopper/blackwell.